geomesa-kafka/geomesa-kafka-datastore/src/test/scala/org/locationtech/geomesa/kafka/EmbeddedZookeeper.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2013-2015 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0 which
+ * accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ */
+
+package org.locationtech.geomesa.kafka
+
+import java.net.InetSocketAddress
+
+import kafka.server.KafkaConfig
+import kafka.utils.{TestUtils, Utils}
+import org.apache.zookeeper.server.{NIOServerCnxnFactory, ZooKeeperServer}
+
+trait HasEmbeddedZookeeper {
```

Comment:
Allow embedded Zookeeper to be used without Kafka.  Maybe with two traits, HasEmbeddedZookeeper for just zookeeper and HasEmbeddedKafka for both.